x/downtime-detector/types/keys.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+import fmt "fmt"
+
+// There are few of these keys, so we don't concern ourselves with small key names.
+var (
+	lastBlockTimestampKey      = []byte("last_block_timestamp")
+	lastDowntimeOfLengthPrefix = "last_downtime_of_length/%s"
```

Comment:
Was the %s supposed to be in the var itself?

Reply:
var itself?

Reply:
Its used in a fmt.Sprintf call later

Reply:
Ah, I have never seen it used this way but it makes sense, thanks :)